From a832ca10c42fa3cc043297e8ff98ae41ae9bfe52 Mon Sep 17 00:00:00 2001
From: Stephane Viau <sviau@codeaurora.org>
Date: Thu, 22 May 2014 10:38:15 -0400
Subject: [PATCH 08/44] drm/msm: rename MSM drm platform driver's name into
 msm_drm

Before this change, the driver's name was defined as "msm". This
definition conflicts with another MSM driver in the downstream
kernel (camera_v2).
To avoid such conflict (kernel crash), the msm drm driver's name
is changed to "msm_drm", which is also a bit more explicit than
the previous definition.

Change-Id: Id8ad18bdea87266fc1f5c87ab2fbc19473612d28
Signed-off-by: Stephane Viau <sviau@codeaurora.org>
---
 drivers/gpu/drm/msm/msm_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/msm/msm_drv.c b/drivers/gpu/drm/msm/msm_drv.c
index 0339c5d..9d57dbf 100644
--- a/drivers/gpu/drm/msm/msm_drv.c
+++ b/drivers/gpu/drm/msm/msm_drv.c
@@ -1152,7 +1152,7 @@ static struct platform_driver msm_platform_driver = {
 	.probe      = msm_pdev_probe,
 	.remove     = msm_pdev_remove,
 	.driver     = {
-		.name   = "msm",
+		.name   = "msm_drm",
 		.of_match_table = dt_match,
 		.pm     = &msm_pm_ops,
 	},
-- 
2.8.2

